internal/store/postgres/metaschema_repository.go

```diff
@@ -77,7 +78,7 @@ func (m MetaSchemaRepository) Get(ctx context.Context, id string) (metaschema.Me
 		case errors.Is(err, sql.ErrNoRows):
 			return metaschema.MetaSchema{}, metaschema.ErrNotExist
 		default:
-			return metaschema.MetaSchema{}, fmt.Errorf("%w: %s", dbErr, err)
+			return metaschema.MetaSchema{}, fmt.Errorf("%w: %s", ErrQueryRun, err)
```

Comment:
lets switch everywhere to use `"%s: %w", err, ErrQueryRun` instead of the opposite